install_manually.md

```diff
@@ -8,7 +8,7 @@ Once you have access to your server, either directly or by SSH, you can install
 
 1. Install git
 ```bash
-sudo apt-get install git
+sudo apt-get install git dialog
```

Comment:
Ok, I didn't found where to install `dialog` on the install script.
I am ok to install it here.